### ISSUE TITLE
Navigation bar overlaps title bar when replacing MainPage for a maui Windows App

### ISSUE THREAD

**Issue by SophisticatedConsulting** [dotnet/maui#10710]
### Description

When an application transfers a login page to an AppShell page the title bar overlaps the Windows title bar:

`            Application.Current.MainPage = new AppShell();
`

<img width="632" alt="image" src="https://user-images.githubusercontent.com/56303860/196022664-b54c3c1f-fa4f-4d17-a693-4126c85b100c.png">

This is a common scenario for many applications.

This has not worked in any previous maui versions or with the current .NET 7 RC1. 

### Steps to Reproduce

1. Start with the maui Flyout Shell demo: [ShellFlyOutSample](https://github.com/dotnet/maui-samples/tree/main/6.0/Navigation/ShellFlyoutSample)
2.  Add a new xaml page named Login.xaml and replace the content:
```
<?xml version="1.0" encoding="utf-8" ?>
<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             x:Class="ShellFlyoutSample.Login"
             Title="Login">
    <ContentPage.Content>
        <StackLayout>
            <Label Text="Please login"
                VerticalOptions="CenterAndExpand" 
                HorizontalOptions="CenterAndExpand" />
            <Button Text="Login" 
                Clicked="Button_Clicked"
                HorizontalOptions="Center" />
        </StackLayout>
    </ContentPage.Content>
</ContentPage>
```
3. Add to the code behind:
```
        private void Button_Clicked(object sender, EventArgs e)
        {
            Application.Current.MainPage = new AppShell();
        }
```
5. Replace the start-up page in the app.xaml.cs code behind:
`		MainPage = new NavigationPage(new Login());
`
6. Run the app as a Windows app. The navbar and title appear correctly for the Login page. When you click on the Login button and replace the main page the navbar and page title overlap window title bar (see screen shot above)

### Link to public reproduction project repository

See above

### Version with bug

6.0.486 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows SDK 10.0.22000.0

### Did you find any workaround?

No.
I tried not extending the app content into the Windows TitleBar (adding the code below to Platform/Windows/app.xaml.cs constructor):

```
        Microsoft.Maui.Handlers.WindowHandler.Mapper.AppendToMapping(nameof(IWindow), (h, v) =>
        {
            (v.Handler.PlatformView as MauiWinUIWindow).ExtendsContentIntoTitleBar = false;
        });
```

This results in the TitleBar being clean but then the page title overlaps the flyout menu hamburger button.

<img width="703" alt="image" src="https://user-images.githubusercontent.com/56303860/196023981-238dc19e-2c95-4a81-b19a-2a2b1231443a.png">

**Comment by PureWeen** [comment]
Duplicate of #10573

**Comment by SophisticatedConsulting** [comment]
Yes... resolved with VS 17.4 Preview 4, MAUI RC2.

**Comment by SophisticatedConsulting** [comment]
Resolved with VS Preview 17.4 Preview 4.0 and MAUI 7.0.0 RC2
Thanks!
